.github/actions/install-node-package/action.yml

```diff
@@ -1,8 +1,10 @@
 name: Install dependencies
+description: Install npm dependencies from package.json
```

Comment:
For me, the name of the action is self-explanatory.

Reply:
Agree; the name is very descriptive.
But without description fields, the file violates [github-action.json](https://json.schemastore.org/github-action.json).

It's valid but schema-checks aren't happy
![grafik](https://github.com/pouchdb/pouchdb/assets/1261013/835cae72-e2ca-4558-94fd-e12bd1eca35c)